### ISSUE TITLE
Macro bar compatablity issue

### ISSUE THREAD

**Issue by ChobitsCrazy** [saif-ellafi/foundryvtt-minimal-ui#139]
Hey thank you for the work on this module!

I'm using Ripper's Taskbar module and it is conflicting with how this module moves the macro bar. I was wondering if it was possible to either A: have the manual option able to be moved with vertical change as well as horizontal. OR B: have an option listed to disable interaction with the macro bar.

As it currently stands if I select the (default) option it still is getting moved to where it normally is instead of letting another module handle the movement of the bar.

![image](https://github.com/saif-ellafi/foundryvtt-minimal-ui/assets/21301107/1cc22692-ef87-442f-a267-f5b2a294956f)

![image](https://github.com/saif-ellafi/foundryvtt-minimal-ui/assets/21301107/b078dd5d-1b8d-4334-b3e2-99801fb19421)

**Comment by cryptiklemur** [comment]
Yeah, if we could just disable the macro hotbar logic of this module, in the settings, that'd be awesome!

**Comment by kaerosd** [comment]
I was coming here for this exactly, having the ability to 'disable' any given set of the minimal UI modifications would be a really nice addition. Just a 'Don't Touch This' for each block of modifications would let me default this back to Taskbar control (or any other set of small UI tweaks).
